python/triton/runtime/autotuner.py

```diff
@@ -134,6 +139,11 @@ def run(self, *args, **kwargs):
         else:
             config = self.configs[0]
         self.best_config = config
+        if self.report and not used_cached_result:
+            autotune_stop = time.time()
+            print(
+                f"Autotuner for function {self.fn} finished after {autotune_stop-autotune_start:.2f}s; best config selected: {self.best_config};"
```

Comment:
Can we use `self.bench_time`?

Reply:
Good point, the hook won't make such a difference. I changed it